src/app/app.module.ts

```diff
@@ -11,6 +11,7 @@ import { AddWordComponent, WordListComponent } from './components';
 import { WordService, ValidationService } from './services';
 import { MultiComponent, ErrorMessageComponent } from './shared/ui-components';
 
+import {NgbModule} from '@ng-bootstrap/ng-bootstrap';
```

Comment:
Is this an official library from angular ?

Reply:
https://ng-bootstrap.github.io/#/getting-started